tensorflow-notebook/config.yaml

```diff
@@ -0,0 +1,5 @@
+jupyter:
+  password: okteto
+
+service:
```

Comment:
@pchico83 @irespaldiza do you think this is something we should just put in the values.yaml instead? If the user changes this, the chart will not be functional.

Reply:
Makes sense, since this is meant to be running in Okteto Cloud

Reply:
I had used config.yaml to try to maintain the official repository chart. As I had to modify it makes sense move it to Values.